docs/hybrid-slurm-cluster/deploy-hybrid-slurm.md

```diff
@@ -0,0 +1,284 @@
+# Deploying and installing the hybrid module
+
+These instructions step you through the deployment, installation and
+verification of a hybrid slurm cluster using the
+[schedmd-slurm-gcp-v5-controller] HPC Toolkit module.
+
+They assume that your on-premise or simulated on-premise slurm
+cluster has already been set up to allow hybrid partitions from the hybrid
+module. If you haven't already, see [hybrid-slurm.md](./hybrid-slurm.md) for
+instructions on preparing your on-premise cluster or
+[inter-gcp-project-hybrid-slurm.md] for
+instructions on preparing a simulated on premise cluster in gcp using Slurm on
+GCP.
+
+> If you came to this page from [inter-gcp-project-hybrid-slurm.md],
```

Comment:
I might make a note that if you are using an on-prem cluster you can ignore the indented sections.